lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -148,6 +148,12 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 		return await tagEntryElement.sendKeys( Key.ENTER );
 	}
 
+	async tagEventuallyDisplayed( tag ) {
+		//const selector = await driverHelper.getElementByText( this.driver, 'span', tag );
```

Comment:
@Stojdza any ideas why this doesn't work? I've tried it locally and it doesn't work

Reply:
Yeah, selector should be `By.css( 'span' )` not just `'span'`. 

`getElementByText()` probably should be more clear about selectors - should we send, for example, `<className>` or `By.css( '.<className>' )`

Reply:
Yeah 🤦‍♂️